src/site/_includes/partials/item.njk

```diff
@@ -17,7 +15,7 @@
   {% elif item %}
     {% if not item.data.draft %}
       <li>
-        <a href="{{ item.url }}">
+        <a href="{{ item.url | getDefaultUrl }}">
```

Comment:
So you add i18n to the item in line 2 url just to remove it here. Can you describe what is the expected effect?